webapp/angular.json

```diff
@@ -81,7 +81,7 @@
                 {
                   "type": "initial",
                   "maximumWarning": "2mb",
-                  "maximumError": "5mb"
+                  "maximumError": "6mb"
```

Comment:
I did some double-checking.  We have been hovering ~4.99mb since at least `4.10` and this finally pushed us over.  

In general we definitely want to avoid anything that significantly increases the bundle size, but it is not really possible to add logic without slightly increasing the size.  So, bumping this another mb to give us more breathing room...